schedule/containers/sle_image_on_sle_ecr.yaml

```diff
@@ -0,0 +1,7 @@
+---
+name: sle_image_on_publiccloud
```

Comment:
I find this naming a bit confusing , all other scenarios named like this meant to show host/guest OS'es . here situation is a bit different SLE ECR is not OS . I would come up with something else . 
Actually this goes together with this comment https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/13202/files#r708267661 . If you accept it this scenario will have name `upload_container_image`